Reference/Management/Models/Content.md

```diff
@@ -212,14 +175,14 @@ var content = contentService.GetById(1234);
 return content.Published;
 ```
 
-### .ReleaseDate
+### .PublishDate
 
 If set, returns `DateTime` indicating when the `Content` should be published and made available on the website and cache.
 
 ```csharp
 // Given a `ContentService` object get Content by its Id and set the release date to 4 days from now
 var content = contentService.GetById(1234);
-content.ReleaseDate = DateTime.Now.AddDays(4);
+return content.ReleaseDate
```

Comment:
ReleaseDate shouldn't be here I guess?

Reply:
@busrasengul You are absolutely right, just updated.